wandb/errors/__init__.py

```diff
@@ -19,31 +22,35 @@ def __init__(self, message, context: Optional[dict] = None) -> None:
         if context:
             self.context = context
 
-    # For python 2 support
-    def encode(self, encoding):
-        return self.message
-
 
 class CommError(Error):
-    """Error communicating with W&B"""
+    """Error communicating with W&B (legacy error for backwards compatibility)"""
 
     def __init__(self, msg, exc=None) -> None:
-        super().__init__(msg)
-        self.message = msg
         self.exc = exc
+        self.message = msg
+        super().__init__(self.message)
 
 
-class UsageError(Error):
-    """API Usage Error"""
+class BackendError(CommError):
+    """Error communicating with W&B backend"""
```

Comment:
Fix this wording, should be errors that communicated from the backend (not necessarily errors communicating with the backend)